docs/develop/showcase.md

```diff
@@ -4,98 +4,80 @@ title: Showcase
 ---
 
 import ThemedImage from '@theme/ThemedImage';
+import LogoText from '@site/src/components/LogoText';
+
 
 An incomplete list of apps, wallets and websites that built on KILT.
-If you are missing from the list and want to be added, you are welcome to open a PR.
+If you are missing from the list and want to be added, you are welcome to [open a PR](https://github.com/KILTprotocol/docs/compare).
```

Comment:
I think it would be better to link to the actual edit link for the showcase.md file: https://github.com/KILTprotocol/docs/edit/develop/docs/develop/showcase.md